src/main/java/com/esri/samples/graphics_update/position_sumulator/UpdateMessageListener.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2021 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.esri.samples.graphics_update.position_sumulator;
+
+import java.util.EventListener;
+
+public interface UpdateMessageListener extends EventListener {
+    /**
+     * Invoked when a vehicle message is generated
+     * @param vehicleUpdateMessageEvent
+     */
+    public void updateMessage(UpdateMessageEvent vehicleUpdateMessageEvent);
```

Comment:
```suggestion
    void updateMessage(UpdateMessageEvent vehicleUpdateMessageEvent);
```

My IDE prompter says: "Modifier 'public' is redundant for interface methods"